### ISSUE TITLE
( Android ) It won't let me open any game 

### ISSUE THREAD

**Issue by SrQubit18** [yuzu-emu/yuzu#10516]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Android first release

### Description of Issue

After configuring everything I get the following error when opening a game:
![Screenshot_20230531-032234](https://github.com/yuzu-emu/yuzu/assets/87690432/b832db75-dc01-4394-8c63-4ad153f2df77)

It tells me to install custom drivers but my gpu is not compatible with that

### Expected Behavior

Just open the game

### Reproduction Steps

Using gpu mali g52 

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/11609434/yuzu_log.txt)

[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/11609438/yuzu_log.txt.old.txt)

Those were the only 2 files in the log folder

### System Configuration

CPU:Unisoc t700
GPU/Driver:mali g52
RAM:4gb
OS:Android 11

**Comment by mishailovic** [comment]
Won't open another issue just because I have another CPU. Same thing happens on my Pixel 6 with Google Tensor. I guess Android version only works with Qualcomm Snapdragon CPUs for now

**Comment by Samsung466** [comment]
Yup same issue with mali G71..hopefully the app gets updated in the future

**Comment by mishailovic** [comment]
Remember that this is literally first android version. I bet things will change dramatically in the near future

**Comment by ransom1122** [comment]
Same thing on Mali-G77

**Comment by FearlessTobi** [comment]
Mali is not supported by us.

**Comment by nathanielcs** [comment]
Only Adreno GPUs are supported atm

**Comment by mishailovic** [comment]
> Only Adreno GPUs are supported atm

Is there any plans on supporting mali gpu?

**Comment by chrtylee** [comment]
> Mali is not supported by us.

Idk what GPU is what but you apparently don't support pixels currently. Is this planned to change? Or should I just uninstall and never try again?
